### ISSUE TITLE
Cant deploy rook cluster

### ISSUE THREAD

**Issue by Sarga** [rook/rook#706]
Hello.

I'm trying to deploy rook with the following config which I grab from the documentation at: https://github.com/rook/rook/blob/master/Documentation/kubernetes.md

I have Kubernetes v1.5.3 cluster

```
apiVersion: rook.io/v1alpha1
kind: Cluster
metadata:
  name: rook
  namespace: rook
spec:
  versionTag: v0.3.1-117-ge01d34f
  dataDirHostPath:
  storage:
    useAllNodes: false
    useAllDevices: false
    deviceFilter:
    metadataDevice:
    location:
    storeConfig:
      storeType: filestore
      databaseSizeMB:
      journalSizeMB:
    nodes:
    - name: "10.10.10.83"
      directories:
      - path: "/ceph/rook"
    - name: "10.10.10.85"
      directories:
      - path: "/ceph/rook"
    - name: "10.10.10.89"
      directories:
      - path: "/ceph/rook"
```

`
Error from server (NotFound): error when creating "rook-cluster.yaml": the server could not find the requested resource
`

**Comment by travisn** [comment]
@Sarga That error is commonly hit if the cluster is created soon after the operator starts. The operator needs 10-20 seconds to initialize the TPRs. If you try again does it work now?

**Comment by Sarga** [comment]
It doesn't working. The operator has already started 6 hours ago. But if I commented section with "nodes", it creates 3 mon's on one node. I think something wrong with yaml file, but I can't find mistake.

```
NAME                                                        READY     STATUS    RESTARTS   AGE
rook-operator-2745552258-q54bs                              1/1       Running   0          6h
```

For now I have raid1 on each node with kubernetes. Cluster with 3 minions and one master
Tell me, please, do I have to a separate disk for the cluster ceph on each node?

**Comment by travisn** [comment]
Three mons on a single node will actually be fixed with #680, I am about ready to merge it. That should not be related to your problem.

If you're seeing the mons created, then it sounds like it is further than I thought. What pods do you have running? And where exactly did you see that error?
```
kubectl -n rook get pods
```
For further discussion, could you join our Gitter? https://gitter.im/rook/rook

**Comment by travisn** [comment]
Please reopen if not seen it fixed with #680
